src/RepoStatus.js

```diff
@@ -219,7 +220,7 @@ export class RepoStatus extends Component {
                 <Row><Col>&nbsp;</Col></Row>
                 <Row>
                     <Col xs={12}>
-                        <CLIEquivalent command="repository status" />
+                        <CliEquivalent command="repository status" />
```

Comment:
if there's a strong reason for this capitalization, can we do `CommandLineEquivalent`, otherwise perhaps revert to `CLIEquivalent`

Reply:
It is bad practice to have abbreviations with more than 2 characters capitalized. I would probably prefer CommandLineEquivalent over Cli anyway, but that is your call. Just don't do `CLIEquivalent`.

Reply:
Golang is fine with those, but when in Rome... I prefer expanding anyway.